docker-compose.yml

```diff
@@ -59,7 +59,7 @@ services:
     volumes:
       - .:/app
       - node_modules:/app/node_modules
-    command: webpack-dev-server --hot --inline --progress --host 0.0.0.0 --config config/webpack.config.js
+    command: yarn webpack
```

Comment:
Actually no, just `yarn webpack` works fine. Do you think that `yarn run webpack` is better?